src/LegendRenderer/SvgOutput.ts

```diff
@@ -9,8 +9,8 @@ export default class SvgOutput extends AbstractOutput {
 
   constructor(
     size: [number, number],
-    maxColumnWidth: number | null,
-    maxColumnHeight: number | null,
+    maxColumnWidth: number | undefined,
```

Comment:
Is this a breaking change?

Reply:
I wouldn't call it breaking as the tests already used undefined as paramater. IMHO its a bugfix as the typing was wrong.